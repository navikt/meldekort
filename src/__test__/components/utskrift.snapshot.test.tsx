import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProviderWrapper from '../utils/providerWrapper';
import Utskrift from '../../app/components/print/utskrift';

it('Sprakvelger', () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <Utskrift />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
