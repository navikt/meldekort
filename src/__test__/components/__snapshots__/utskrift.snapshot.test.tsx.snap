// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sprakvelger 1`] = `null`;
